modules/n1ql/pages/n1ql-language-reference/typefun.adoc

```diff
@@ -93,6 +97,14 @@ SELECT ISATOM(true) AS `boolean`,
 
 === Description
 
+Type functions perform operations that check or convert expressions.
+
+=== Arguments
+
+expression:: [Required] The expression to check.
+
+=== Return Value
+
 Returns True if expression is a Boolean, otherwise returns MISSING, NULL or false.
 
 === Examples
```

Comment:
The result of the example is still wrong. Look at the description of this function: we would expect that it will return `true` for `boolean`; `null` for `null`; and `false` for everything else.